source/release-notes/1.12.txt

```diff
@@ -0,0 +1,52 @@
+.. _c2c-release-notes-1.12:
+
+================================
+Release Notes for mongosync 1.12
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page describes changes and new features introduced in  
+{+c2c-full-product-name+} 1.12.
+
+.. _1.12.0-c2c-release-notes:
+
+1.12.0 Release
+--------------
+
+**February 19, 2025**
+
+``enableUserWriteBlocking`` parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``enableUserWriteBlocking`` parameter in the 
+:ref:`/start <c2c-api-start>` API now accepts the following string options:
+
+- ``"sourceAndDestination"`` for dual write-blocking
+- ``"none"`` for no write-blocking
+
+See :ref:`c2c-api-start-params`. 
+
+Live Upgrades
+~~~~~~~~~~~~~
+
+:ref:`Live upgrades <c2c-versioning-live-upgrade>` 
+to version 1.12 are not supported.
```

Comment:
This content is from https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/611, but copied in wholesale since that one is pending final LGTM before it can be merged.